### PR TITLE
tests for lwc reservationHelperForm

### PR DESCRIPTION
**Pull request by msrivastav13**
### What does this PR do?

tests for lwc reservationHelperForm

### What issues does this PR fix or reference?

#<Insert GitHub Issue>

## The PR fulfills these requirements:

[ ] Tests for the proposed changes have been added/updated.
[ ] Code linting and formatting was performed.

### Functionality Before

<insert gif and/or summary>

### Functionality After

<insert gif and/or summary>

### CODE REVIEW

File: es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                    'c-customer-detail-form'
+                );
+                customerDetailForm.dispatchEvent(
+                    new CustomEvent('customerupdate', {
+                        detail: STATE_NEW
+                    })
+                );
+            })
+            .then(() => {
+                Promise.resolve().then(() => {
```
Comment (pozil) [inline es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js]: Extra `Promise.resolve()` is not needed

File: es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                    'c-reservation-detail-form'
+                );
+                reservationDetailForm.dispatchEvent(
+                    new CustomEvent('draftreservation', {
+                        detail: EVENTDETAILS
+                    })
+                );
+            })
+            .then(() => {
+                Promise.resolve().then(() => {
```
Comment (pozil) [inline es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js]: Extra `Promise.resolve()` is not needed

**Review by pozil — CHANGES_REQUESTED** [review]

File: es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js (hunk 1 above)
Comment (msrivastav13) [reply es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js]: done! I thought that once event is dispatched it's asynchronous and needs to resolve! However, I see it works.

File: es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js (hunk 2 above)
Comment (msrivastav13) [reply es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js]: done!

File: es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js (hunk 1 above)
Comment (msrivastav13) [reply es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js]: https://github.com/trailheadapps/easy-spaces-lwc/commit/c30b02af8ee98863a264abe80992b9c4d9b9268a

File: es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js (hunk 2 above)
Comment (msrivastav13) [reply es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js]: https://github.com/trailheadapps/easy-spaces-lwc/commit/c30b02af8ee98863a264abe80992b9c4d9b9268a

Any idea why my recent commit does not show up in the PR? seems bizarre!

Comment (msrivastav13) [reply es-space-mgmt/main/default/lwc/reservationHelperForm/__tests__/reservationHelperForm.test.js]: ignore previous it showed after some time!

**Comment by msrivastav13** [comment]
@pozil please reReview this!

**Review by pozil — APPROVED** [review]

**Pull request merged**